### PR TITLE
Development

### PR DESCRIPTION
**Pull request by shailscript**
Please review this as soon as possible.

### CODE REVIEW

**Review by tonyfeung — COMMENTED** [review]
should remove unwanted files from commit, such as: .DS_Store

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/underdev.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <div class=backimg>
+        <span class="noise"></span>
```
Comment (tonyfeung) [inline index.html:13]: remove below unused block of code

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import credentials from './credentials.js'
+
+
+document.querySelector('form').addEventListener('submit', function(e) {
+    e.preventDefault();
+    let email = document.querySelector('input#search').value
+    console.log(email)
+    var studentInfo = credentials.studentsInfo.filter(stu => {
+        return stu.email = email
```
Comment (tonyfeung) [inline js/app.js]: '===' instead of '='?

Comment (tonyfeung) [inline js/app.js]: as I see `js/credentials.js`, is it wrong field name? should be 'stu.student_email' instead
```suggestion
        return stu.student_email === email
```

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                let email = document.createElement('span')
+                let school = document.createElement('span')
+                let depID = document.createElement('span')
+                let courseCode = document.createElement('span')
+                let status = document.createElement('span')
+                let grade = document.createElement('span')
+
+                certificate.className = 'certificate'
+                name.innerHTML = `<strong>Name :</strong> ${item.first_name} ${item.last_name}`
+                email.innerHTML = `<strong>Email :</strong> ${item.email}`
```
Comment (tonyfeung) [inline js/app.js]: correct field name
```suggestion
                email.innerHTML = `<strong>Email :</strong> ${item.student_email}`
```

File: js/app.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import credentials from './credentials.js'
+
+
+document.querySelector('form').addEventListener('submit', function(e) {
+    e.preventDefault();
+    let email = document.querySelector('input#search').value
+    console.log(email)
```
Comment (tonyfeung) [inline js/app.js]: remove console.log line

File: js/app.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                school.innerHTML = `<strong>School :</strong> ${item.school}`
+                depID.innerHTML = `<strong>Department ID :</strong> ${item.department_id}`
+                courseCode.innerHTML = `<strong>Course Code :</strong> ${item.Course_code}`
+                status.innerHTML = `<strong>Status :</strong> ${item.Status}`
+                grade.innerHTML = `<strong>Grade :</strong> ${item.Grade}`
+
+                certificate.append(name ,email, school, depID, courseCode, status, grade)
+                resultContainer.append(certificate)
+            })
+        }
```
Comment (tonyfeung) [inline js/app.js:40]: nice to have friendly messages if there is no student found

**Review by tonyfeung — COMMENTED** [review]
@mkdesign I just made some comments, pls take a look.

**Review by tonyfeung — CHANGES_REQUESTED** [review]
@mkdesign I just made some comments, pls take a look.

**Review by tonyfeung — APPROVED** [review]

**Review by Shivani912 — APPROVED** [review]

**Pull request merged**